plugin.xml

```diff
@@ -13,13 +13,23 @@
         <dependency id="cordova-plugin-intercom" version="^6.2.0" />
 
         <config-file target="AndroidManifest.xml" parent="/manifest/application">
-            <service android:name="com.zenput.push.FirebaseMessagingRouterService">
+            <service android:name="com.zenput.push.FirebaseMessagingRouterService",
+                     android:stopWithTask="false">
```

Comment:
This prevents the services from being stopped when the app is removed from the recent apps list. [Source](https://stackoverflow.com/a/40035807/2603945)